cmd/vela-worker/exec.go

```diff
@@ -60,6 +62,19 @@ func (w *Worker) exec(index int, config *library.Worker) error {
 		return err
 	}
 
+	// request build executable containing pipeline.Build data using exec client
+	execBuildExecutable, resp, err := execBuildClient.Build.GetBuildExecutable(item.Repo.GetOrg(), item.Repo.GetName(), item.Build.GetNumber())
```

Comment:
following the golangci failure, if we dont need `resp` can we omit the variable? 
```suggestion
	execBuildExecutable, _, err := execBuildClient.Build.GetBuildExecutable(item.Repo.GetOrg(), item.Repo.GetName(), item.Build.GetNumber())
```

Reply:
I updated the code with that change